src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -0,0 +1,129 @@
+package nextstep.subway.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Embeddable
+public class Sections {
+    private static final Integer MIN_SECTION_COUNT = 1;
+    private static final String REMOVE_ERROR_COUNT = "구간 삭제 실패 - 구간이 %d 개 이하입니다.";
+    private static final String REMOVE_ERROR_LAST = "구간 삭제 실패 - 마지막 구간만 제거할 수 있습니다.";
+    @OneToMany(mappedBy = "subwayLine", cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
상수와 인스턴스 변수의 구분을 위해 개행을 추가해보면 좋을 것 같아요.
```suggestion
    private static final String REMOVE_ERROR_LAST = "구간 삭제 실패 - 마지막 구간만 제거할 수 있습니다.";
    
    @OneToMany(mappedBy = "subwayLine", cascade = CascadeType.ALL, orphanRemoval = true)
```